canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.scss

```diff
@@ -66,6 +79,13 @@ $properties-vt-checkbox-width: 45px;
 				}
 			}
 		}
+
+		&:hover {
+			background-color: rgba($brand-02, 0.2);
```

Comment:
Is this the color from design?

Reply:
Table header background color on hover - `$active-ui`
Resizing line color - `$active-secondary`
Ref - https://ibm-analytics.slack.com/archives/CQ378CWSJ/p1647554721031409?thread_ts=1643400352.963079&cid=CQ378CWSJ